lib/iex/endpoints/historial_prices.rb

```diff
@@ -0,0 +1,21 @@
+module IEX
+  module Endpoints
+    module HistoricalPrices
+      def historical_prices(symbol, options = {})
+        slug = "stock/#{symbol}/chart"
+        slug += "/#{options[:range]}" if options[:range]
+        slug += "/#{options[:date]}" if options[:range] == 'date'
```

Comment:
This will be wrong if `options[:date]` is missing. I would add a check to `raise ArgumentError` on a missing date when range is date or something like that. Make sure to add specs for the combinations.

Reply:
Technically this will just return an empty array but that's pretty useless so I'll add the argument error. 

Good call about the specs, pretty sloppy of me. While adding specs I noticed that you can actually receive a different payload if you ask for a specific date and don't include additional query params (ex: chartByDay: true). This will take a bit of time to investigate how to best solve so It'll be a bit before this PR is ready for review again.

Reply:
Don't let this block you.

Reply:
Sounds good. Basically you can get 2 types of data from the IEX endpoint, per day or per min data. The current implementation only supports the per day format.